Add Sidecar's Asset Hub public instances (#5102)
w3f/polkadot-wiki: docs/build/build-integrate-assets.md

- Please refer to [docs](https://paritytech.github.io/substrate-api-sidecar/dist/) for full usage
  information. Details on options like how to make a historical query are not included here.

> There are also available the following public instances:
> 
> - [Sidecar connected to Polkadot Asset Hub](https://polkadot-asset-hub-public-sidecar.parity-chains.parity.io)
>   and
> - [Sidecar connected to Kusama Asset Hub](https://kusama-asset-hub-public-sidecar.parity-chains.parity.io)
> 
> The purpose of these instances is to give the opportunity to anyone to quickly check and get an
> overview of the info that the asset related endpoints provide.
> 
> :::caution
> 
> These instances should only be used for ad-hoc checks or tests and not for production, heavy testing
> or any other critical purpose.
> 
> :::
> 

Reviewer: ```suggestion
Here are the available public instances:

- [Sidecar connected to Polkadot Asset Hub](https://polkadot-asset-hub-public-sidecar.parity-chains.parity.io)
  and
- [Sidecar connected to Kusama Asset Hub](https://kusama-asset-hub-public-sidecar.parity-chains.parity.io)

The purpose of these instances is to allow anyone to check and get a quick
overview of the info that the asset-related endpoints provide.

:::caution

These instances should only be used for ad-hoc checks or tests and not for production, heavy testing
or any other critical purpose.

:::

```
